lib/common/util.js

```diff
@@ -138,13 +138,25 @@ exports.publishChange = publishChange;
 
 
 /*
- * Poll a job until it reaches either the succeeded or failed state.
+ * Poll a job until it:
+ * - reaches either the succeeded or failed state
+ * - completes a given task
+ *
+ * 'opts' is an options object whose properties can be:
+ * - 'taskName': a string that identifies the nane of the task to wait for
+ * before calling the callback 'cb'. If 'opts.taskName' is set, then pollJob
+ * will not wait for the whole job to succeed or fail, but instead will just
+ * wait for that specific task to complete.
  *
  * Note: if a job fails, it's the caller's responsibility to check for a failed
  * job.  The error object will be null even if the job fails.
  */
 
 function pollJob(opts, cb) {
+    assert.object(opts, 'opts must be an object');
```

Comment:
With assert-plus I think in general you don't want to put a useful string as the second argument. I found this weird too, but it seems the normal thing to do is:

```
assert.object(opts, 'opts');
```

because:

```
> var assert = require('assert-plus')
undefined
> opts = 'beer';
'beer'
> assert.object(opts, 'opts must be an object');
AssertionError: opts must be an object (object) is required
    at repl:1:8
    at REPLServer.self.eval (repl.js:110:21)
    at repl.js:249:20
    at REPLServer.self.eval (repl.js:122:7)
    at Interface.<anonymous> (repl.js:239:12)
    at Interface.emit (events.js:95:17)
    at Interface._onLine (readline.js:203:10)
    at Interface._line (readline.js:532:8)
    at Interface._ttyWrite (readline.js:761:14)
    at ReadStream.onkeypress (readline.js:100:10)
> assert.object(opts, 'opts');
AssertionError: opts (object) is required
    at repl:1:8
    at REPLServer.self.eval (repl.js:110:21)
    at repl.js:249:20
    at REPLServer.self.eval (repl.js:122:7)
    at Interface.<anonymous> (repl.js:239:12)
    at Interface.emit (events.js:95:17)
    at Interface._onLine (readline.js:203:10)
    at Interface._line (readline.js:532:8)
    at Interface._ttyWrite (readline.js:761:14)
    at ReadStream.onkeypress (readline.js:100:10)
>
```